metrics.go

```diff
@@ -15,10 +15,20 @@ var (
 		Help: "A gauge representing the current status of remote services schemas",
 	})
 
-	promServiceUpdateError = prometheus.NewCounterVec(
+	promServiceUpdateErrorCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
+			Name: "service_update_failures",
+			Help: "A counter indicating how many times services have failed to update",
+		},
+		[]string{
+			"service",
+		},
+	)
+
+	promServiceUpdateError = prometheus.NewGaugeVec(
```

Comment:
Would rename `s/promServiceUpdateError/promServiceUpdateErrorGauge` so it's the same as the counter.